tests/test_vb.py

```diff
@@ -18,8 +18,8 @@ def jacobian(self, parameters):
         return np.vstack([d_dm, d_dc]).T
 
 
-class ModelError(VariationalBayesInterface):
-    def __init__(self, forward_model, data):
+class ModelError:
+    def __init__(self, forward_model, data, with_jacobian):
```

Comment:
could you test that automatically (if method is implemented, call it, otherwise don't?)

Reply:
Your idea would also work. I don't know what is better and would leave it as is, to save some work/time :)